discovery-infra/test_infra/helper_classes/kube_helpers/global_vars.py

```diff
@@ -3,7 +3,6 @@
 # TODO - Temporary import - for backward compatibility
 from test_infra.consts.kube_api import *
 
-# TODO - remove all env variables dependency
```

Comment:
Leave that one on.
On https://issues.redhat.com/browse/MGMT-5347 it will be solved